frontends/ol-search-ui/assets/learning-resource-card.scss

```diff
@@ -43,7 +43,16 @@ $smallFontSize: 0.75em !default;
       margin-top: 0; 
     }
     &:last-child {
-      margin-bottom: 0; 
+      margin-bottom: 0;
+      /*
+      Last child of ol-lrc-content will take up any extra space (flex: 1) but
+      with its contents at the bottom of its box.
+      */
+      flex: 1;
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-end;
+      align-items: flex-start; // The default is stretch, we we do not want.
```

Comment:
This improves the alignment of card content...

Before:

<img width="760" alt="Screen Shot 2022-08-15 at 10 41 51 AM" src="https://user-images.githubusercontent.com/9010790/184656962-174dfd3b-08b6-418b-8315-b6f1de5dcc4c.png">

Now:
<img width="807" alt="Screen Shot 2022-08-15 at 9 51 07 AM" src="https://user-images.githubusercontent.com/9010790/184657000-49c117d9-b1ba-495a-a198-0ab9b633913b.png">